service/src/test/java/bio/terra/workspace/app/controller/ReferencedGcpResourceControllerGcsBucketConnectedTest.java

```diff
@@ -138,14 +155,8 @@ public void update() throws Exception {
             newDescription,
             newBucketName,
             newCloningInstruction);
-    // Update the sourceResource to the updated one as all the tests are sharing
-    // the same resource.
-    ApiGcpGcsBucketResource getResource =
-        mockMvcUtils.getReferencedGcsBucket(
-            userAccessUtils.defaultUserAuthRequest(),
-            workspaceId,
-            sourceResource.getMetadata().getResourceId());
-    assertEquals(updatedResource, getResource);
+    bucketIsUpdated = true;
```

Comment:
nit: this should be set right before the `updateReferencedGcsBucket` call instead to prevent the tiny edge case where the call causes a test failure but still makes the update happen